run.sh

```diff
@@ -120,6 +120,9 @@ case "$deployment_action" in
     "clean_airship")
         clean_airship
         ;;
+    "gather_logs")
```

Comment:
Do we really need a user interface for this?

Reply:
yes. If Im deploying locally and it fails during deploy_osh I want to be able to gather all logs, because I might not be able to troubleshoot it in that moment, so its good to expose this

Reply:
I am fine with that, I just love playing devil's advocate :D